src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -979,6 +979,7 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
             int position,
             SyntaxToken tokenOnLeftOfPosition,
             CancellationToken cancellationToken,
+            ISet<SyntaxKind> allowedModifiersBefore = null,
```

Comment:
@CyrusNajmabadi I don't understand that comment. Are you saying you don't care to offer `this` after `ref`? this is the whole purpose of touching the IDE in this PR.

---
In reply to: [120768720](https://github.com/dotnet/roslyn/pull/19703#discussion_r120768720) [](ancestors = 120768720)